tox.ini

```diff
@@ -1,8 +1,10 @@
 [tox]
-envlist = csslint,eslint,pycodestyle,pylint,py35-django{111,20,21,22}
+envlist = csslint,eslint,pycodestyle,pylint,py35-django{22,30}
```

Comment:
shouldn't we add python 3.8 here as well?

Reply:
You are right. I've added it.